app/assets/stylesheets/application.css

```diff
@@ -17,19 +17,63 @@
  .background img {
    z-index: -1;
    position:fixed;
-   opacity:0.8;
+   opacity:0.5;
    transform: scaleX(-1);
    background-size: cover;
  }
  .content {
    font-family:"Arial";
+
+
    position: absolute;
+   width: 80vw;
+   height: 100%;
    top:0;
+
    margin-top: 80px;
+   margin-left: 10vw;
+   margin-right: 10vw;
+
+   padding: 10px;
  }
- .content {
-   margin-left: 20px;
+ .content-left{
+   background-color:#134234;
+   color:#DCFFCF;
+   position: absolute;
+   width: 40vw;
+   height: 80%;
+   top:0;
+   left: 0;
+
+   margin-top: 80px;
+   padding: 10px;
+   border-left: solid 8px #0077A6;
+ }
+ .content-right{
+   background-color:#185946;
```

Comment:
content right is a section that displays content on right of screen

Reply:
We dont mess around out here